### ISSUE TITLE
Crosscompile for AARCH64 with neon fails

### ISSUE THREAD

**Issue by N30dG** [FFTW/fftw3#60]
Hello,
I'm trying to make a X-Compile from debian 8 to AARCH64 with neon using aarch64-linux-gnu-gcc. Without NEON I have no problem's. But with setting the --enable-neon flag it fails.

Here is my build-log:
[fftw-build.txt](https://github.com/FFTW/fftw3/files/211783/fftw-build.txt)

I really don't know what I'm doing wrong.

**Comment by matteo-frigo** [comment]
The first error is this:

/usr/lib/gcc/aarch64-linux-gnu/4.9/include/arm_neon.h: In function 'vqadd_u8':
/usr/lib/gcc/aarch64-linux-gnu/4.9/include/arm_neon.h:2116:3: error: incompatible types when returning type 'int' but 'uint8x8_t' was expected
   return __builtin_aarch64_uqaddv8qi_uuu (__a, __b);

I have no idea what this is, but it doesn't seem to have anything to do with us.  I.e., a header file declares a function that tries to cast uint8x8_t into int.  You'll have to debug your gcc installation, I am afraid.

**Comment by N30dG** [comment]
> You'll have to debug your gcc installation, I am afraid.<

I don't know how to do this. My gcc comes from Debian-Repository. It's simply Installed using apt-get ;)

**Comment by dfarns** [comment]
What version of gcc are you using?  Perhaps your version of GCC is insufficient for the 64 bit neon intrinsics?  I think gcc 4.9.3 or later is sufficient, but i don't know if the 4.9.1 arm neon header is new enough.

**Comment by N30dG** [comment]
Yes you are right! using GCC 5.3 works fine :). Before I used GCC 4.9.2.
Can be closed, thanks :)
